app/templates/sidebar.html

```diff
@@ -16,6 +16,22 @@
           My Profile
         </a>
       </li>
+    {% endif %} 
+    {% if g.current_user.isFaculty %}
+    <li>
+      <a href="/profile/{{g.current_user}}"
+         class="nav-link text-white {{'active' if 'profile' in request.path }}"
+         {{"aria-current='page'" if 'profile' in request.path }}>
+        My Profile
+      </a>
+    </li>
```

Comment:
Since this is identical to lines 12-18 above, we should think about just including 'my profile' for everyone. I'm leaning towards yes. That would get it out of the 'if' statements